### DISCUSSION TITLE
Styles from node_modules library not included.

### DISCUSSION THREAD

**Discussion by OllieJT** [tailwindlabs/tailwindcss#7837] [Help] [answered]
### What version of Tailwind CSS are you using?
```json
{
	"@tailwindcss/forms": "^0.5.0",
	"@tailwindcss/typography": "^0.5.2",
	"prettier-plugin-tailwindcss": "^0.1.8",
	"tailwindcss": "^3.0.23",
}
```

### What build tool (or framework if it abstracts the build tool) are you using?

```json
{
	"@sveltejs/kit": "next",
	"autoprefixer": "^10.4.2",
	"postcss": "^8.4.8",
	"postcss-import": "^14.0.2",
	"svelte": "^3.46.4",
	"svelte-preprocess": "^4.10.4",
}
```

### What version of Node.js are you using?

v16.13.2

### What browser are you using?

Latest Chrome (13 Mar 2022)

### What operating system are you using?

macOS

### Reproduction / Describe your issue

I can't provide an exact reproduction as it requires an external UI library.

Example:
I have an npm package with one component:
```html
<!--/Tag.svelte-->
<span class="inline-flex items-center p-3 text-xs font-medium bg-blue-100 text-blue-800">
    <slot />
</span>
```

I import and use the component into my active project

```html
<script>
    import Tag from "my-npm-package/Tag.svelte";
</script>

<!-- Using my imported component does not work -->
<Tag> Hello! </Tag>
```

However the component does not render as expected. Only the tailwind classes that I am using elsewhere in my project are styling the component from my library.

If I simply paste the raw code from the component next to the imported component - they both render just fine:

```html
<script>
    import Tag from "my-npm-package/Tag.svelte";
</script>

<span class="inline-flex items-center p-3 text-xs font-medium bg-blue-100 text-blue-800">
    Hello!
</span>

<!-- Now that we're using the required styles in this app - the imported component works. -->
<Tag> Hello! </Tag>
```

This leads me to believe the JIT compiler doesn't know to include the styles from my external component. 

I don't see any documentation on this and so assume it is a bug.

### Things I've tried...

Pasting the raw code into my application. This works but defeats the point of the package.

included the package in the tailwind config:

```js
const config = {
	content: [
		"./src/**/*.{html,js,svelte,ts}",
		"my-component/**/*.{html,js,svelte,ts}",
		"/node_modules/my-component/**/*.{html,js,svelte,ts}",
	],
}
```

**Comment by aggreggator** [comment, marked as answer]
Try to add a dot prepending your path.

**Reply by adamwathan** [reply]
Yep, you need something more like this:

```diff
  const config = {
    content: [
      "./src/**/*.{html,js,svelte,ts}",
      "my-component/**/*.{html,js,svelte,ts}",
-     "/node_modules/my-component/**/*.{html,js,svelte,ts}",
+     "./node_modules/my-component/**/*.{html,js,svelte,ts}",
    ],
  }
```

**Reply by OllieJT** [reply]
Thank you both!
